src/tests/sunsynk/test_helpers.py

```diff
@@ -96,7 +105,23 @@ def test_time() -> None:
     assert time.minutes == just_before_midnight
 
 
+def test_pack_unpack() -> None:
+    """Test pack_value and unpack_value functions."""
+    # Test 16-bit values
```

Comment:
In test_rwsensors, the sensor is not signed (that test case needs fixing factor to -1)

Can we add a test here to show we expect it to fail?

```python
wiih pytest.raises(struct.error):
  pack_value(-1, bits=16, signed=False)
```